packages/chevrotain/test/parse/recognizer_spec.ts

```diff
@@ -113,7 +113,7 @@ function defineRecognizerSpecs(
 
           public subRule = this.RULE(
             "subRule",
-            (numFromCaller, charFromCaller) => {
+            (numFromCaller?: number, charFromCaller?: string) => {
```

Comment:
are these parameters optional to avoid a compile error?

Reply:
yes